.changeset/three-yaks-clean.md

```diff
@@ -0,0 +1,5 @@
+---
+"@itwin/itwinui-css": minor
```

Comment:
Is the `itwinui-css` change just a patch since no API changes have happened?

```suggestion
"@itwin/itwinui-css": patch
```

Also, `itwinui-react` would also need to be mentioned since that is also indirectly affected. You could create a new changeset for the react with a react specific message like:

```
"@itwin/itwinui-react": minor

Removed `margin-block` from `Badge`.
```

Reply:
Lets quickly release a patch for both and seal off itwinui-css 2.x (via #1910)

```suggestion
"@itwin/itwinui-css": patch
"@itwin/itwinui-react": patch
```

Reply:
Changed to `patch` for both CSS & React.

Reply:
Since #1910 is now merged, these will need to be split into separate changesets, per Rohan's suggestion.